configs/resnet/metafile.yml

```diff
@@ -0,0 +1,217 @@
+Collections:
+  - Name: ResNet
+    Metadata:
+      Training Data: ImageNet
+      Training Techniques:
+        - SGD with Momentum
+        - Weight Decay
+      Training Resources: 8x V100 GPUs
+      Epochs: 100
+      Batch Size: 256
+      Architecture:
+        - ResNet
+    Paper: https://openaccess.thecvf.com/content_cvpr_2016/html/He_Deep_Residual_Learning_CVPR_2016_paper.html
+    README: configs/resnet/README.md
+
+Models:
+- Config: configs/resnet/resnet18_b16x8_cifar10.py
+  In Collection: ResNet
+  Metadata:
+    FLOPs: 0.56 G
+    Parameters: 11.17 M
+    Training Data: Cifar-10
+    Training Resources: 8x 1080 GPUs
+    Epochs: 200
+    Batch Size: 128
+  Name: ResNet-18-b16x8
```

Comment:
name could use similar name as config, thus it can be distinguished easier and will not be duplicated.